Benchmarks/Benchmarks/PackageGraphBenchmarks/PackageGraphBenchmarks.swift

```diff
@@ -4,11 +4,20 @@ import PackageModel
 import Workspace
 
 let benchmarks = {
+    let defaultMetrics: [BenchmarkMetric] = [
+        .mallocCountTotal,
+        .syscalls,
+    ]
+
     Benchmark(
-        "Package graph loading",
+        "PackageGraphLoading",
```

Comment:
Working around a known `package-benchmark` issue with whitespace handling in benchmark names when reading recorded thresholds.